helm_express/helm-express/values.yaml

```diff
@@ -83,3 +87,12 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+# Redis
+redis:
+  image: redis:6.2-alpine
+  containerPort: 6379
+# Mongo
+mongo:
+  image: mongo:5.0.9
+  containerPort: 27017
```

Comment:
Helm chart use `values.yaml` to keep these customized param. Therefore, we can publish our `chart` for people to deploy the same infra with their own param (like image version etc.)